lib/rollbar/capistrano_tasks.rb

```diff
@@ -123,6 +123,10 @@ def skip_in_dry_run(logger, dry_run)
       end
 
       def info_request_response(logger, result)
+        # TODO: Is this logger output wanted or expected?
+        # Should there be a switch to disable?
+        return
```

Comment:
See related github issue for discussion about this logger output. It looks like it includes output that needs to be scrubbed, and maybe should be debug level when turned on.